OMCompiler/Compiler/Script/CevalScriptBackend.mo

```diff
@@ -3209,6 +3211,9 @@ algorithm
   end match;
 end getAdjacencyMatrix;
 
+/* -------------------------------------------------------------------
+                         RUN OLD FRONTEND
+   ------------------------------------------------------------------- */
```

Comment:
Remove or add this as a docstring instead:) 
""
Runs the old frontend. Does X Y Z. Returns XX YY ZZ etc etc
""

Reply:
what is special about these comments?

Reply:
I think they are used to auto generate documentation. Typically docstrings are used for functions wheras /**/ are used within them